uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -1227,6 +1227,122 @@ Feature: GGMQ-1
       | mqtt-v | name        | agent                                       | recipe                  |
       | v5     | paho-python | aws.greengrass.client.Mqtt5PythonPahoClient | client_python_paho.yaml |
 
+  @GGMQ-1-T22
+  Scenario Outline: GGMQ-1-T22-<mqtt-v>-<name>: As a customer, I can send a message of size 128KiB to the MQTT broker
```

Comment:
this test should validate that messages larger than 128KB can be sent/received using the local broker. The 128KB limit is the limit for IoT Core, but not for local messages which do not go to iot core.

Reply:
Logic of GGMQ-1-T22 as well as scenario description are the same as in original GGAD-1-T22:

Large messsage is sent to local MQTT broker by a client, forwarded to IoT Core by Bridge component and received from IoT Core by the same client.
As for me no issues here, we testing components as wide as possible.

Reply:
I'd like you to add more steps to this test. The steps should send messages _locally_ which are larger than 128KB.

Reply:
The steps were added in 8ccc2d619b6aa64506a2dd3380adb6987d088bcd